libopae/src/properties.c

```diff
@@ -492,7 +521,53 @@ fpgaPropertiesSetObjectType(fpga_properties prop, fpga_objtype objtype)
 	return result;
 }
 
-fpga_result __FPGA_API__ fpgaPropertiesGetBus(const fpga_properties prop, uint8_t *bus)
+fpga_result __FPGA_API__ fpgaPropertiesGetSegment(const fpga_properties prop,
+		uint32_t *seg)
+{
+	struct _fpga_properties *_prop = (struct _fpga_properties *)prop;
+	fpga_result result = FPGA_OK;
+	int err = 0;
+
+	ASSERT_NOT_NULL(seg);
+	result = prop_check_and_lock(_prop);
+	if (result)
+		return result;
+
+	if (FIELD_VALID(_prop, FPGA_PROPERTY_SEGMENT)) {
+		*seg = _prop->segment;
+	} else {
+		FPGA_MSG("No segment");
+		result = FPGA_NOT_FOUND;
+	}
+
+	err = pthread_mutex_unlock(&_prop->lock);
+	if (err)
+		FPGA_ERR("pthread_mutex_unlock() failed: %s", strerror(err));
+	return result;
+}
+
+fpga_result __FPGA_API__ fpgaPropertiesSetSegment(fpga_properties prop,
```

Comment:
where is the declaration for fpgaPropertiesSetSegment and fpgaPropertiesGetSegment